docs/messages_guide/src/asciidoc/_chapters/binder_msgs.adoc

```diff
@@ -1861,13 +1848,13 @@ incompatible data types.
 == SQL 4128
 
 ```
-Default volume and subvolume information could not be retrieved fromc=_DEFAULTS define - DEFINEINFO error <number>.
+Default volume and sub volume information could not be retrieved from=_DEFAULTS define - DEFINEINFO error <number>.
```

Comment:
There are bunches of those. I don't know which FS errors are and are not
used in Trafodion so I hope someone else can bring out the eraser...


On Mon, Feb 1, 2016 at 10:09 AM, DaveBirdsall <notifications@github.com>
wrote:

> In docs/messages_guide/src/asciidoc/_chapters/binder_msgs.adoc
> <https://github.com/apache/incubator-trafodion/pull/294#discussion_r51447198>
> :
>
> > @@ -1861,13 +1848,13 @@ incompatible data types.
> >  == SQL 4128
> >
> >  ```
> > -Default volume and subvolume information could not be retrieved fromc=_DEFAULTS define - DEFINEINFO error <number>.
> > +Default volume and sub volume information could not be retrieved from=_DEFAULTS define - DEFINEINFO error <number>.
>
> I would think subvolume would be correct, however, this message is
> Guardian-specific and should never occur in Trafodion.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/294/files#r51447198>.
>



-- 
Thanks,

Gunnar
*If you think you can you can, if you think you can't you're right.*